qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -362,7 +362,7 @@ def get_snap_stats(self, dir_path):
         log.debug(f'snapshots: {snapshots}');
 
         result = self.fs_snap_schedule_cmd('status', path=dir_path,
-                                           snap_schedule='1M', format='json')
```

Comment:
Why `snap_schedule='1M'` removed?

Reply:
As the commit message says, this turned out to be a spurious argument for the **`snap-schedule status`** command when the subvol interface was removed.
Since the `subvol` argument was never used in the `status` command implementation, the `1M` argument was effectively dormant. Now that the `subvol` formal paramater requirement has been removed, the `1M` argument took the place of an argument that was optional but used in the `snap-schedule status` command implementation, which turns out to be invalid.